plugin/PDFDetailsPage.qml

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2024 Damien Caliste
+ * Contact: Damien Caliste <dcaliste@free.fr>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2 only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Office 1.0
+
+DetailsPage {
+    DetailItem {
+        //: Page count of the PDF document
+        //% "Page Count"
+        label: qsTrId("sailfish-office-la-pdf-pagecount")
+        value: document.pageCount
+        alignment: Qt.AlignLeft
+        visible: !document.locked
+    }
+    DetailItem {
+        //% "Read protection"
+        label: qsTrId("sailfish-office-la-pdf-readprotection")
+        value: document.passwordProtected
+            //% "by password"
```

Comment:
This looks sort of strange since password is the only way to protect the document. Also the "none" string is effectively a dead translation so thus could be better just avoided.

How about we do this a bit different altogether. Tried quickly here something like:
```

    SectionHeader {
        text: "Read-protection"
    }

    Label {
        width: parent.width - 2*x
        x: Theme.horizontalPageMargin
        text: "This document is protected by a password"
        color: palette.secondaryHighlightColor
        font.pixelSize: Theme.fontSizeSmall
        textFormat: Text.PlainText
        wrapMode: Text.Wrap
    }

```

Which didn't seem too bad. The password field and button maybe needing some more padding around them.

Reply:
Very nice suggestion. I've updated the page accordingly. I've added a paddingMedium between the text and the password field.